test/dlc_tests/container_tests/bin/pytorch_tests/testPyTorchS3

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Please add the invocation of this script to the `test/dlc_tests/sanity` tests, because it doesn't need to be run on an EC2 instance, does it?

Reply:
Moved to sanity test